packages/platforms/mellanox/x86-64/x86-64-mlnx-msn2700/onlp/builds/src/module/auto/x86_64_mlnx_msn2700.yml

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_mlnx_msn2700 Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_mlnx_msn2700_CONFIG_INCLUDE_LOGGING:
```

Comment:
These configuration defines are by convention all uppercase. 
Please change to X86_64_MLNX_MSN2700_XXX and rerun the autogenerator.